bot/discord/events/message.js

```diff
@@ -1,12 +1,20 @@
 const fetch = require('node-fetch');
 const axios = require('axios');
 
-const blacklistedWords = [
-    'rape', 'nigga', 'nigger', 'jew'
-]
-
 module.exports = (client, message) => {
-    if (blacklistedWords.includes(message.content.toLowerCase())) { message.delete(), message.reply('Do __NOT__ use that word in this server. You will get muted next time...') }
```

Comment:
please return if the author is a bot @Darker-Ink

Reply:
gonna kick the bot instead returning is a bad idea since bots could then just spam the nword